tests/e2e/kubetest2-kops/deployer/deployer.go

```diff
@@ -48,13 +48,14 @@ type deployer struct {
 	KopsBaseURL          string `flag:"-"`
 	PublishVersionMarker string `flag:"publish-version-marker" desc:"The GCS path to which the --kops-version-marker is uploaded if the tests pass"`
 
-	ClusterName    string   `flag:"cluster-name" desc:"The FQDN to use for the cluster name"`
-	CloudProvider  string   `flag:"cloud-provider" desc:"Which cloud provider to use"`
-	GCPProject     string   `flag:"gcp-project" desc:"Which GCP Project to use when --cloud-provider=gce"`
-	Env            []string `flag:"env" desc:"Additional env vars to set for kops commands in NAME=VALUE format"`
-	CreateArgs     string   `flag:"create-args" desc:"Extra space-separated arguments passed to 'kops create cluster'"`
-	KopsBinaryPath string   `flag:"kops-binary-path" desc:"The path to kops executable used for testing"`
-	createBucket   bool     `flag:"-"`
+	ClusterName      string   `flag:"cluster-name" desc:"The FQDN to use for the cluster name"`
+	ControlPlaneSize int      `flag:"control-plane-size" desc:"Number of control plane instances"`
```

Comment:
Can we use `Master` instead of `ControlPlane`? I don't think we use it anywhere else in kOps.